app/views/DeploymentHistoryPage.scala.html

```diff
@@ -162,3 +155,33 @@ <h1>Deployments</h1>
     <a href="/deployments/@{env.asString}" class="navbar-link" id="@{env.asString}-tab" onclick="javascript:selectEnvironment(event, '@{env.asString}')" role="tab">@env.displayString</a>
 </li>
 }
+
+@pageNavigation() = {
+  <tr>
+      <td>
+          @if(pagination.page <= 0) {
+              <a disabled="true" class="btn btn-default" href="#">Prev</a>
+          }else {
+              <a class="btn btn-default" href="@{Pagination.uriForPage(request.uri, pagination.page-1)}">Prev</a>
```

Comment:
formatting with a space around operators makes it clear that page-1 is not just kebab-case